var/spack/repos/builtin/packages/ccache/package.py

```diff
@@ -12,13 +12,14 @@ class Ccache(AutotoolsPackage):
     again."""
 
     homepage = "https://ccache.samba.org/"
-    url      = "https://www.samba.org/ftp/ccache/ccache-3.3.4.tar.gz"
+    url      = "https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3.7.7.tar.gz"
```

Comment:
```suggestion
    url      = "https://github.com/ccache/ccache/releases/download/v3.7.1/ccache-3.7.1.tar.gz"
```
Also, can you confirm that all of the old releases still download and have the same checksum from this new URL?